### DISCUSSION TITLE
Umbraco 9RC: ILocalizedTextService.Localize fallback to current thread culture?

### DISCUSSION THREAD

**Discussion by enkelmedia** [umbraco/Umbraco-CMS#10642] [Features and ideas]
When migrating "The Dashboard"-package to V9 I found that ILocalizedTextService.Localize has a new parameter for CultureInfo.

Not sure if I'm using this in the right way but this means I constantly have to pass the current culture, 

```csharp
_localizedTextService.Localize("theDashboard","publishedContentNodes", Thread.CurrentThread.CurrentCulture),
```

Would be nice to have an overload that would set the culture to the current Threads culture if not passed?

**Comment by nul800sebastiaan** [comment]
That's what `public static string Localize<T>(this ILocalizedTextService manager, string area, T key)` does? 

https://github.com/umbraco/Umbraco-CMS/blob/v9/dev/src/Umbraco.Core/Services/LocalizedTextServiceExtensions.cs#L19
